main.cpp

```diff
@@ -51,85 +51,107 @@ arm.flex();                             // 3)
 arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
 
 
-//  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
-//  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
-//  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
-//  4)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
-//  5)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
-//  6)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
-//  7)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
-//  8)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  1)
+//  Noun: piston
+//  action 1: piston releases
+piston.release();
+//  action 2: piston retracts
+piston.retract();
+//  action 3: piston prepares to fire
+piston.arm();
+
+//  2)
+//  Noun: microphone
+//  action 1: microphone listens for audio
+microphone.listenAudio();
+//  action 2: microphone gets phantom power
+microphone.phantomPower();
+
+//  action 3: microphone coverts audio
+microphone.convertAudio();
+
+//  3)
+//  Noun: pizza
+//  action 1: pizza is made
+pizza.make();
+
+//  action 2: pizza gets toppings
+pizza.getToppings();
+
+//  action 3: pizza gets cooked
+pizza.cook();
+
+//  4)
+//  Noun: trash can
+//  action 1: trash can filled with trash
+trashCan.fill();
+//  action 2: trash can is empty
+trashCan.empty();
+
+//  action 3: trash can is put on curb
+trashCan.putOnCurb();
+
+//  5)
+//  Noun: school
+//  action 1: school hires teachers
+school.hireTeacher().
+//  action 2: school buys chairs
+school.buyChair();
+
+//  action 3: school gives knowdlege
+school.givesKnowledge();
+
+//  6)
+//  Noun: Waiter
+//  action 1: waiter serves a table
+waiter.servesTable();
+
+//  action 2: waiter gets orders
+waiter.getOrder();
+
+//  action 3: waiter gives the bill
+waiter.giveBill();
+
+//  7)
+//  Noun: mechanic
+//  action 1: mechanic fixes car
+mechanic.fixCar();
+
+//  action 2: mechanic bills me
+mechanic.billCurrentCustomer();
+//  action 3: mechanic gets paid
+mechanic.getPaid();
```

Comment:
As explained in the instructions, here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
REPLACE the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.

Do not create an 11th noun.

Once you have completed the above-requested changes, please do the following:
- click each checkbox to mark each requested change as completed.
- click the `[Resolve Conversation]` button.
- Re-request a review of your pull request. It's on the right side of the Pull request page where you originally requested my review.
- Send me the link to your pull request in a new message.